site/docs/v1/tech/messengers/message-templates.adoc

```diff
@@ -0,0 +1,102 @@
+---
+layout: doc
+title: Message Templates
```

Comment:
Hmmm. I apologize. I think we duplicated some effort here: https://fusionauth.io/docs/v1/tech/email-templates/message-templates/

Can you add a link to that message templates doc here and remove this file?

I would love for you to add the additional replacement variables to that section. I wasn't able to find and test all those, but if you have, please add them.